### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by tomassedovic**
#### What does this PR do?

This fixes the sample inventory and readme to work out-of-the-box.

#### How should this be manually tested?

Following the readme step by step should now work.

#### Is there a relevant Issue open for this?

There is no issue for this, these are trivial fixes.

#### Who would you like to review this?
cc: @redhat-cop/casl

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,22 +47,28 @@ Cool! Now you're ready to provision OpenShift clusters
...
 ```
 
+**Note**: If you see an error saying that there are no `config/*.sh` files, try
+temporarily disabling SELinux (`sudo setenforce 0`) and running a new
+container.
+
 3) Run the `end-to-end` provisioning playbook
 ```
-ansible-playbook -i /root/repository/casl-ansible/inventory/sample.casl.example.com.d/inventory /root/repository/casl-ansible/playbooks/openshift/end-to-end.yml
+ansible-playbook -i /root/repository/casl-ansible/inventory/sample.casl.example.com.d/inventory /root/repository/casl-ansible/playbooks/openshift/end-to-end.yml -e openstack_ssh_public_key=openshift
```
Comment (oybed) [inline README.md]: could we make the added param something like `-e openstack_ssh_public_key=<your_ssh_key_name>`  to indicate that they need to supply whatever name they have for their key? The comment below is good, but would like to make the example a bit more generic as well.

File: inventory/sample.casl.example.com.d/inventory/group_vars/all.yml [hunk 2]
```diff
@@ -7,28 +7,36 @@ ansible_become: True
...
+# Using Red Hat Satellite:
+rhsm_satellite: 'sat-6.example.com'
+rhsm_org: 'CASL_ORG'
+rhsm_activationkey: 'casl-latest'
 rhsm_password: ''
 
+# # Or using RHN username, password and optionally pool:
+# rhsm_username: ''
+# rhsm_password: ''
+# rhsm_pool: ''  # leave commented out if you want to `--auto-attach` a pool
```
Comment (oybed) [inline inventory/sample.casl.example.com.d/inventory/group_vars/all.yml]: I believe we have seen issues with comment at the end of the line. When uncommenting this line, the user need to know to remove the additional "# leave commented ..." portion as well. This often doesn't happen, hence, I'd like to recommend that we move the additional comment above the actual variable line. 

These are great additions to the sample inventory - thank you for adding them.

**Comment by oybed** [comment]
LGTM  - @etsauer do you want to review? Otherwise I can merge.

File: README.md [hunk 3]
```diff
@@ -47,22 +47,28 @@ Cool! Now you're ready to provision OpenShift clusters
...
 
 The following is just an example on how the `sample.casl.example.com` inventory can be used:
 
-1) Edit `casl-ansible/inventory/sample.casl.example.com/inventory/hosts` and edit the `# OpenStack Provisioning Variables` and `# Subscription Management` sections at the top to match your environment/project/tenant. See comments in the file for more detailed information on how to fill these in.
+1) Edit `casl-ansible/inventory/sample.casl.example.com.d/inventory/hosts` and edit the `# OpenStack Provisioning Variables` and `# Subscription Management` sections at the top to match your environment/project/tenant. See comments in the file for more detailed information on how to fill these in.
 
 2) Start the `openstack-client` container.
 ```
-./casl-ansible/docker/openstack-client-centos/run.sh --repository=/path/to/repos/dir/
+docker run -it --name control-host -v $HOME/.ssh:/root/.ssh -v $HOME/.config/openstack:/root/.config/openstack -v $HOME/src:/root/repository -v $HOME/.ansible.cfg:/root/.ansible.cfg docker.io/redhatcop/control-host-openstack bash
```
Comment (etsauer) [inline README.md]: we actually have a docker-compose file now that makes this simpler. should be:

```
docker-compose up -d
docker exec -it <container-name> bash
```

**Comment by etsauer** [comment]
@tomassedovic just the one comment. then i'm good.

**Comment by tomassedovic** [comment]
@etsauer done

**Comment by etsauer** [comment]
LGTM

**Pull request merged**